packages/cojson/src/coValueCore/coValueCore.ts

```diff
@@ -181,8 +181,9 @@ export class CoValueCore {
   get verified() {
     return this._verified;
   }
-  private readonly peers = new Map<
-    PeerID,
+
+  private readonly loadingStatuses = new Map<
+    string,
```

Comment:
Why stop using PeerID?

Reply:
Because this also accepts "storage" and PeerID is just string anyway.

But good point, I think being explicit helps on documenting what's in there so changing into `PeerID | "storage"`